addons/prometheus/template/values.yaml

```diff
@@ -0,0 +1,2421 @@
+# Default values for kube-prometheus-stack.
```

Comment:
are these all defaults? wanna add comments where there are changes?

Reply:
can do - there's a fair few of them

Reply:
there's a bunch of changes
I'll start adding comments